core/src/main/thrift/MLSchema.thrift

```diff
@@ -76,6 +76,12 @@ struct ModelHeader {
   // Multiclass labels.
   7: optional list<LabelDictionaryEntry> labelDictionary;
   8: optional map<string, list<double>> labelEmbedding;
+  // The number of hidden layers in neural network
+  9: optional i32 numHiddenLayers;
+  // the activation function in neural network
+  10: optional string activationFunction;
```

Comment:
I am thinking to start with a global activation function, but I can change this to support per-layer activation function